app/services/prison_seeder/seed_entry.rb

```diff
@@ -48,6 +48,14 @@ def name
     hash.fetch('name')
   end
 
+  def closed
+    hash.fetch('closed', false)
```

Comment:
Don't need the default here since we already have a default in the db, plus a not null constraint.